packages/upstream/x509.0.11.2/opam

```diff
@@ -33,7 +33,7 @@ depends: [
   "alcotest" {with-test}
   "cstruct-unix" {with-test & >= "3.0.0"}
   "mirage-crypto-rng" {with-test}
-  "mirage-crypto-pk" {with-test & < "0.8.9"}
+  "mirage-crypto-pk" {with-test}
```

Comment:
The version check is there because of an RSA issue that was fixed in 0.8.9, I'm not sure we can relax this

Reply:
removed this